sphinx/source/start_network.rst

```diff
@@ -73,6 +73,32 @@ Other members are then allowed to vote for the proposal, using the proposal ID r
 
 The user is successfully added once a :term:`quorum` of members have accepted the proposal (``"result":true"``).
 
+Register the Lua application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: This section only applies when deploying Lua applications (i.e. using the ``libluageneric.so.signed`` enclave library). For C++ applications, this step should be skipped.
+
+Before opening the CCF network to users, members should vote to register the Lua application defining the user-specific business logic (see for example :ref:`Logging (Lua)`):
+
+.. code-block:: bash
+
+    $ memberclient --cert member1_cert --privk member1_privk --rpc-address rpc_ip:rpc_port --ca network_cert update_lua_app --lua-app-file /path/to/lua/app_script
+    {"commit":9,"global_commit":8,"id":0,"jsonrpc":"2.0","result":{"completed":false,"id":1},"term":2}
+
+Other members are then allowed to vote for the proposal, using the proposal ID returned to the proposer member (here ``1``, as per ``"result":{"completed":false,"id":1}``).
+
+.. code-block::
```

Comment:
Should we display the step where the member queries for the proposal and maybe show what it would look like?

Reply:
There's already a specific page in the docs for governance operations: https://microsoft.github.io/CCF/governance.html
`start_network.rst` is just about starting a network so should focus on that IMO. There's already quite a few links to the governance page from there as well.